shared-plugins/vip-go-elasticsearch/vip-go-elasticsearch.php

```diff
@@ -126,7 +126,7 @@ public function filter__the_posts( $posts, $query ) {
 		if ( ! $query->is_main_query() || ! $query->is_search() )
 			return $posts;
 
-		if ( ! is_array( $this->search_result ) )
+		if ( is_wp_error( $this->search_result ) || ! is_array( $this->search_result ) || empty( $this->search_result['results'] ) || empty( $this->search_result['results']['hits'] ) )
```

Comment:
The `$this->search_result` may be an object of WP_Error ( see https://github.com/Automattic/vip-go-mu-plugins/blob/930f369d80be9d75eeb83b4c22a24e774880fb41/shared-plugins/vip-go-elasticsearch/vip-go-elasticsearch.php#L282 ) and thus treating the object as an array would produce PHP Fatal error.